### DISCUSSION TITLE
Improvements to editing files

### DISCUSSION THREAD

**Discussion by NashJames** [community/community#42329] [Mobile]
A couple small things to make the UX a bit better:
- Hide the keyboard by default when editing files/issues/comments. It takes up half the screen.
- There's no option for hiding the keyboard on discussions/issues/replies. Only files. Either add a button or the normal, at least on iOS, is to be able to drag your finger down the middle of the screen to hide it.
- The text seems larger when editing? Add an option to make or smaller, or even better, pinch to zoom out.
- Obviously, a preview option is very useful as well.

As weird as it is to say, it's a better experience using the website to edit stuff still. It's got a lot more freedom of movement and less of the screen is taken up by fixed buttons. I say this hoping the app becomes a stronger UX than the website. Thanks for all your work.

Edit: I only just noticed the change text size option in the settings. But this only affects editing files? And the lowest option is the only useful one? The lower bound should be at least a couple options smaller than the regularly useful size.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone 11 (iOS 15.6)
</details>

**Comment by jfullstackdev** [comment]
that's a good observation, the mobile browser is the GitHub Web itself, so you experience the same thing

though, as for me, I'm not using phone as a productivity tool and I don't have a phone, 

I just have a tablet (which I just won from a Christmas raffle 5 years ago, which is basically Android), the only interest to me back then was to explore everything in Android OS as a developer ( file systems, FTP , security vulnerabilities etc. )
